### PR TITLE
Adding BSC receipts

### PR DESCRIPTION
**Pull request by spiderjako**

### CODE REVIEW

**Review by antisaa — APPROVED** [review]

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        "node": true,
+        "commonjs": true,
+        "es2021": true,
+	"mocha": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": 12
+    },
+    "rules": {
```
Comment (YordanPavlov) [inline .eslintrc.js:12]: Where do those rules come from? Is this some convention?

File: bin/run_bnb.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+ #! /bin/sh
+export DRY_RUN=1
```
Comment (YordanPavlov) [inline bin/run_bnb.sh]: Since the docker-compose has a local Kafka, I think the default mode should not be DRY_RUN

**Review by YordanPavlov — APPROVED** [review]
This exporter should eventually be moved to the common 'san-chain-exporter' so if we settle on some style conventions, we should implement there. Also I think we would get most return on our time if we migrate to TypeScript.

File: .eslintrc.js (hunk 1 above)
Comment (spiderjako) [reply .eslintrc.js:12]: https://eslint.org/docs/latest/rules/

Comment (YordanPavlov) [reply .eslintrc.js:12]: I mean why we settled on:
```
 "quotes": ["error", "double"],
  "semi": ["error", "never"]
```
seems the opposite of what Google Style guide for example suggests:
https://google.github.io/styleguide/jsguide.html#formatting-semicolons-are-required

I don't think we have a company-wide conventions for this.

**Pull request merged**